Checkout/Checkout/Source/Validation/Validators/CardNumberValidator.swift

```diff
@@ -44,7 +44,7 @@ final class CardNumberValidator: CardNumberValidating {
   }
     
   func validateCompleteness(cardNumber: String) -> Result<ValidationScheme, ValidationError.CardNumber> {
-    let cardNumber = cardNumber.filter { Int("\($0)") != nil }
+    let cardNumber = cardNumber.filter { !$0.isWhitespace }
 
     guard validateDigitsOnly(in: cardNumber) else {
```

Comment:
I notice duplicate `.filter { !$0.isWhitespace }` 
I suggest to make String extension to remove white space and replaced in all Frames
```suggestion
 guard validateDigitsOnly(in: cardNumber.removeWhiteSpaces)
```